### PR TITLE
fix geojson response to always be a feature collection

### PR DESCRIPTION
**Pull request by C-Loftus**
Always return a feature collection instead of one feature if there is only one. 
Clean up tests to reflect this.
Handle a few edge cases where no data is returned.

### CODE REVIEW

File: rise/lib/geojson/types.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+class GeojsonFeature(TypedDict):
+    type: Literal["Feature"]
+    geometry: dict
+    properties: dict
+    id: int
+
+
+class GeojsonResponseDict(TypedDict):
```
Comment (webb-ben) [inline rise/lib/geojson/types.py:14]: Could also name this `GeojsonFeatureCollection`
```suggestion
class GeojsonFeatureCollection(TypedDict):
```

File: rise/lib/location.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
 import shapely
 import shapely.wkt
 from rise.env import TRACER
+from rise.lib.geojson.types import GeojsonResponseDict
```
Comment (webb-ben) [inline rise/lib/location.py:12]:
```suggestion
from rise.lib.geojson.types import GeojsonFeatureCollection
```

File: rise/lib/location.py [hunk 3]
```diff
@@ -239,7 +240,7 @@ def to_geojson(
             str, dict[Literal["type"], Literal["number", "string", "integer"]]
         ] = {},
         sortby: Optional[list[SortDict]] = None,  # now treat as list[SortDict]
-    ) -> dict:
+    ) -> GeojsonResponseDict:
```
Comment (webb-ben) [inline rise/lib/location.py:243]:
```suggestion
    ) -> GeojsonFeatureCollection:
```

File: rise/lib/location.py [hunk 4]
```diff
@@ -332,7 +330,7 @@ def sort_key(f):
         validated_geojson = FeatureCollection(
             type="FeatureCollection", features=geojson_features
         )
-        return validated_geojson.model_dump(by_alias=True)
+        return GeojsonResponseDict(**validated_geojson.model_dump(by_alias=True))
```
Comment (webb-ben) [inline rise/lib/location.py:333]:
```suggestion
        return GeojsonFeatureCollection(**validated_geojson.model_dump(by_alias=True))
```

File: rise/rise.py [hunk 5]
```diff
@@ -50,7 +51,7 @@ def items(
         offset: Optional[int] = 0,
         skip_geometry: Optional[bool] = False,
         **kwargs,
-    ):
+    ) -> GeojsonResponseDict:
```
Comment (webb-ben) [inline rise/rise.py:54]:
```suggestion
    ) -> GeojsonFeatureCollection:
```

File: rise/rise.py [hunk 6]
```diff
@@ -8,6 +8,7 @@
 from pygeoapi.util import crs_transform
 from rise.env import TRACER
 from rise.lib.cache import RISECache
+from rise.lib.geojson.types import GeojsonResponseDict
```
Comment (webb-ben) [inline rise/rise.py:11]:
```suggestion
from rise.lib.geojson.types import GeojsonFeatureCollection
```

**Comment by C-Loftus** [comment]
Renames should be addressed

**Review by webb-ben — APPROVED** [review]
